Changelog.md

```diff
@@ -2,10 +2,13 @@
 
 Language Features:
  * Wasm backend: Add ``i32.ctz``, ``i64.ctz``, ``i32.popcnt``, and ``i64.popcnt``.
+ * Inheritance: Allow overrides to be more strict than their base, excluding ``payable``.
```

Comment:
```suggestion
 * Inheritance: Allow overrides to have stricter state mutability: ``view`` can override ``nonpayable`` and ``pure`` can override ``view``.
```
(I think it is not too clear what stricter means)
(we can also only have last, explicit part)